EKF/mag_fusion.cpp

```diff
@@ -623,14 +623,14 @@ void Ekf::fuseHeading()
 		}
 	}
 
-	// wrap the heading to the interval between +-pi
-	measured_hdg = wrap_pi(measured_hdg);
```

Comment:
Does this mean there is a bug in the wrap_pi function? If so then the function should be fixed.